app/components/MarkdownViewer.tsx

```
@@ -44,106 +49,211 @@ const RiskItem: React.FC<{ risk: Risk }> = ({ risk }) => {
+    <div className="grid grid-cols-2 gap-4 h-screen">
+      <div className="bg-[#1A1A1A] text-white p-4 rounded-md overflow-y-auto">
+        {loading && <FaSpinner data-testid="spinner" className="text-4xl animate-spin" />}
+        {error && <p>{error}</p>}
+        {!loading && !error && risks.length === 0 && <p>No risks found.</p>}
+        {!loading && risks.map((risk, index) => <RiskItem key={index} risk={risk} />)}
+      </div>
+      <div className="bg-gray-800 text-white p-4 rounded-md overflow-y-auto">
+        <h3 className="text-lg font-semibold mb-2">AI Response:</h3>
+        <ReactMarkdown
+          children={aiResponse || "Waiting for AI response..."}
+          remarkPlugins={[remarkGfm, remarkBreaks]}  // Add remark-breaks here
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)